css/common.css

```diff
@@ -0,0 +1,70 @@
+body {
```

Comment:
charset の指定を追加してもらえればこちらOKです〜👍
```suggestion
@charset 'utf-8' ;

body {
```